gvsbuild/info.py

```diff
@@ -0,0 +1,10 @@
+import typer
+
+
+def version(active: bool):
```

Comment:
The name of your function is causing your variable name to be shadowed. Perhaps rename the function to `version_callback` or similar would prevent this. 👍